Jenkinsfile

```diff
@@ -107,7 +119,7 @@ pipeline {
         versioner(
           token: "${GITHUB_PACKAGES_TOKEN}"
         , dry: true
-        , repo: repo
+        , repo: REPO
```

Comment:
Can you add `, branch: "master"` to the `versioner` blocks? The new version understands that and it'll set us up to change over to using `main` branch whenever we're ready